### DISCUSSION TITLE
Is possible to call API update/create, without triggering webhook

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#3016] [Archive]
Sometimes I need to update data via API without triggering the webhook. Maybe we can have an option when we create the webhook to enable this on the `app`, `api`, or both. 

Another idea is to have another param that we can bypass to item `create` or `update` via API that we can tell to call the webhook or not.

What do you think?

**Comment by rijkvanzanten** [comment]
That's what Actions is for:

![CleanShot 2020-11-09 at 10 32 56](https://user-images.githubusercontent.com/9141017/98561486-f3668680-2276-11eb-9080-c8873a6032ff.png)

That allows you to only fire your webhook on create/update/delete or a combination of those

**Reply by rijkvanzanten** [reply]
If you need more flexibility than that, I recommend using a regular hook extension so you can fine tune when to do what 🙂

**Comment by botzill** [comment]
Yes but I mean that from the UI i want to trigger webhook all th time, but
from my backend when I update some field, i don't want to trigger webhook
at all. You see?

**Reply by botzill** [reply]
I fix this by using a custom User-Agents for my API calls and I know when a new webhook call is from the API, I can ignore that, if from the app I continue.
